fixtures/storybook-config/src/StorybookDecorator.tsx

```diff
@@ -2,9 +2,15 @@ import 'braid-design-system/reset';
 import { BraidProvider } from 'braid-design-system';
 import apac from 'braid-design-system/themes/apac';
 import React from 'react';
+// @ts-expect-error
```

Comment:
What's the error here? No types?

Reply:
Yep. Seek style guide only ever had flow types. Could be a `@ts-ignore` but it doesn't really matter in this case.

Reply:
In that case, might be worth leaving a comment

```suggestion
// @ts-expect-error no types
```